tdd-seed/src/main/java/org/example/stringCalculator/CalculatorMethod.java

```diff
@@ -0,0 +1,43 @@
+package org.example.stringCalculator;
+
+import java.util.Optional;
+
+public class CalculatorMethod {
+    public CalculatorMethod() {
+    }
+    // 연산을 위한 메서드
+    public int arithmeticResult(int firstNum, char operatorSymbol, int secondNum) {
+        if (operatorSymbol == '+') return add(firstNum, secondNum);
+        if (operatorSymbol == '-') return sub(firstNum, secondNum);
+        if (operatorSymbol == '/') return div(firstNum, secondNum);
+        if (operatorSymbol == '*') return mul(firstNum, secondNum);
```

Comment:
```suggestion
        if (operatorSymbol == '+') {
            return add(firstNum, secondNum);
        }
        if (operatorSymbol == '-') { 
            return sub(firstNum, secondNum);
        }
        if (operatorSymbol == '/') {
            return div(firstNum, secondNum); 
        }
        if (operatorSymbol == '*') {
            return mul(firstNum, secondNum);
        }
```
중괄호 생략은 코드 누락을 유발할 수 있어요
가독성도 좋지만, 유지보수 차원에서 있는게 더 나을 것 같아요

Reply:
조건에 한줄은 괄호 생략하는게 더 좋은건가 생각했었는데 다시 생각해보겠습니다 !

Reply:
수정완